__docs__/wonder-blocks-labeled-field/labeled-field.stories.tsx

```diff
@@ -0,0 +1,165 @@
+import * as React from "react";
+import type {Meta, StoryObj} from "@storybook/react";
+import ComponentInfo from "../../.storybook/components/component-info";
+import packageConfig from "../../packages/wonder-blocks-labeled-field/package.json";
+import {LabeledField} from "@khanacademy/wonder-blocks-labeled-field";
+import {TextArea, TextField} from "@khanacademy/wonder-blocks-form";
+import LabeledFieldArgTypes from "./labeled-field.argtypes";
+import {PropsFor, View} from "@khanacademy/wonder-blocks-core";
+import {spacing} from "@khanacademy/wonder-blocks-tokens";
+import {
+    MultiSelect,
+    OptionItem,
+    SingleSelect,
+} from "@khanacademy/wonder-blocks-dropdown";
+import SearchField from "@khanacademy/wonder-blocks-search-field";
+
+/**
+ * The `LabeledField` component provides common elements for a form field such
+ * as the label, required indicator, description, and error message.
+ *
+ * It is highly recommended that all form fields should be used with the
+ * `LabeledField` component so that our form fields are consistent and accessible.
+ */
+export default {
+    title: "Packages / LabeledField / LabeledField",
+    component: LabeledField,
+    parameters: {
+        componentSubtitle: (
+            <ComponentInfo
+                name={packageConfig.name}
+                version={packageConfig.version}
+            />
+        ),
+    },
+    argTypes: LabeledFieldArgTypes,
+} as Meta<typeof LabeledField>;
+
+type StoryComponentType = StoryObj<typeof LabeledField>;
+
+export const Default: StoryComponentType = {
+    args: {
+        field: <TextField value="" onChange={() => {}} />,
+        label: "Name",
+        description: "Helpful description text",
+        error: "Error message",
+        required: true,
+    },
+};
+
+const AllFields = (args: PropsFor<typeof LabeledField>) => {
```

Comment:
**praise:** Nice, it's great that you are reusing this for all the stories. Also great to include all the possible variations here.